src/main/resources/project/forms/deployment/CreateVMForm.xml

```diff
@@ -159,16 +159,14 @@
     <formElement>
         <label>Resource workspace:</label>
         <property>resource_workspace</property>
-        <value/>
+        <value>default</value>
         <documentation>If you specify a resource pool name, this is the workspace that will be assigned to the created resources.</documentation>
```

Comment:
Same with workspace. This will automatically be defaulted to default by the system if none is specified.